src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+	<include resource="org/springframework/boot/logging/logback/defaults.xml"/>
+	<include resource="org/springframework/boot/logging/logback/console-appender.xml"/>
+
+	<root level="INFO">
+		<appender-ref ref="CONSOLE"/>
+	</root>
+</configuration>
```

Comment:
Does the Spring recognise the logback file with `-test` postfix by default?

Reply:
Yes this is a cherry pick from v23

Reply:
#885 is the original one